test/rowHeight.test.ts

```diff
@@ -21,37 +20,24 @@ function setupGrid(rowHeight: DataGridProps<Row>['rowHeight']) {
   setup({ columns, rows, rowHeight });
 }
 
-test('rowHeight is number', async () => {
+test('rowHeight is number', () => {
   setupGrid(40);
 
   const grid = screen.getByRole('grid');
-  expect(grid).toHaveStyle({ 'grid-template-rows': 'repeat(1, 40px) repeat(50, 40px)' });
-  expect(getRows()).toHaveLength(31);
-
-  await userEvent.tab();
-  expect(grid.scrollTop).toBe(0);
-
-  // Go to the last cell
-  const spy = vi.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
-  await userEvent.keyboard('{Control>}{end}');
-  expect(spy).toHaveBeenCalled();
```

Comment:
Should we keep this?
We still want to check that we can navigate and scroll to cells outside the current viewport, regardless of whether we use a static or dynamic row height.

Reply:
We can reduce the number of rows then.